### PR TITLE
FUSETOOLS2-2402: Setup code coverage

### PR DESCRIPTION
**Pull request by pospisilf**
coverage_1
coverage_2

### CODE REVIEW

File: .vscodeignore [hunk 1]
```diff
@@ -15,3 +15,10 @@ sonar-project.properties
 test Fixture with speci@l chars
 test-resources
 .test-extensions
+camel-ls-sbom.json
+docs
+coverage
+.camel-jbang
+node_modules
```
Comment (apupier) [inline .vscodeignore:22]: to verify: i think that node_modules is mandatory in the vsix file

**Comment by apupier** [comment]
Can you share the results in the PR to see if it is coherent or not?

File: .vscodeignore (hunk 1 above)
Comment (pospisilf) [reply .vscodeignore:22]: This was inspired from LSP coverage PR - https://github.com/camel-tooling/camel-lsp-client-vscode/pull/1755/files#diff-0a721cd4753ff50bbbe1237397c3c7692080254fa766268fcc8b05ef633c50bfR28

Comment (apupier) [reply .vscodeignore:22]: this is what i remembered, need to reinclude it and do another release: https://github.com/camel-tooling/camel-lsp-client-vscode/commit/f9cd25dff7ed457efa1a0a0255b07b0003824436

Comment (pospisilf) [reply .vscodeignore:22]: ok, my bad - I overlooked this commit, will remove

**Review by apupier — APPROVED** [review]

**Review by djelinek — APPROVED** [review]

**Pull request merged**